### PR TITLE
Revamped multi-context section

### PR DESCRIPTION
**Pull request by domaincomponents**
Revamped Multi-Contexts section

- Visual representation of multi-contexts
- Expanded Command line Capabilities for Context maintenance
- Mention of UI console and REST API capabilities for Context maintenance

### CODE REVIEW

File: axon-server/administration/backup-and-messaging-only-nodes.md [hunk 1]
```diff
@@ -1,59 +1,132 @@
-# Backup and Messaging only Nodes
+# Backup and Messaging-only Nodes
```
Comment (smcvb) [inline axon-server/administration/backup-and-messaging-only-nodes.md:1]: Don't these file changes belong to [this](https://github.com/AxonIQ/reference-guide/pull/132) PR you've provided?

File: axon-server/administration/multi-context.md [hunk 2]
```diff
@@ -2,17 +2,138 @@
...
+
+A depiction of multiple registered contexts within an Axon Server EE cluster is shown below
+
+![Multiple contexts within an Axon Server EE cluster](../../.gitbook/assets/multi-context.jpg)
+
+The [clustering](../installation/axon-server-clustering.md) section details the creation of the \__admin and default contexts_ when a new Axon Server cluster is created. The _\_admin_ context __is used to process all configuration changes in Axon Server, so it contains the master configuration from which all contexts get their information. The __\__admin_ context cannot be deleted and thus not visible under the data directory. The _default_ context is the context used by clients when they have not specified any context information. In case you would like to create a cluster without creating a default context, it is recommended to use the [Automatic-Initialization](../installation/axon-server-clustering.md#automatic-initialization) feature where you can control explicitly which contexts can be created or not.
```
Comment (smcvb) [inline axon-server/administration/multi-context.md:13]: `context cannot be deleted and thus _is_ not visible under the data directory.` i.o. `context cannot be deleted and thus not visible under the data directory.`

File: axon-server/administration/multi-context.md [hunk 3]
```diff
@@ -2,17 +2,138 @@
...
+The delete-node-from-context command helps in the deletion member node from an existing context. A sample of the command with the mandatory parameters is depicted below
+
+```
+$ java -jar ./axonserver-cli.jar delete-node-from-context -S http://[node]:[port] -c [context-name] -n [node name]‌
+```
+
+_Mandatory parameters_
+
+* _**-c**_ refers to an existing context
+* _**-n**_ refers to the node name that should be a member of this context
```
Comment (smcvb) [inline axon-server/administration/multi-context.md:103]: Wouldn't it make more sense to adjust this sentence to this:
`* _**-n**_ refers to the node name that should _no longer_ be a member of this context`
?

File: axon-server/administration/backup-and-messaging-only-nodes.md [hunk 4]
```diff
@@ -1,59 +1,132 @@
...
+When you have larger clusters and specific requirements, you may want to have nodes acting in a different way within a context. For this reason, in Axon Server Enterprise 4.3, we have introduced a number of new roles that can be assigned to a node.
 
-* Primary node, this is the role for nodes prior to 4.3. Primary nodes handle client connections and messages, store events
+These are:
 
-  and may act as the leader for a context.
+* _Primary nodes -&gt;_ This is the role for nodes prior to 4.3. Primary nodes handle client connections and messages, store events and may act as the leader for a context.
+* _Backup nodes -&gt;_ These nodes maintain a copy of the event store, but will never become the leader within an [Axon Server cluster](../installation/axon-server-clustering.md). There are two types of Backup nodes - [Active Backup](backup-and-messaging-only-nodes.md#active-backup-node) and [Passive Backup](backup-and-messaging-only-nodes.md#passive-backup-node)
```
Comment (MGathier) [inline axon-server/administration/backup-and-messaging-only-nodes.md:12]: backup nodes never become leader in a context (not in cluster)

File: axon-server/administration/multi-context.md (hunk 2 above)
Comment (MGathier) [reply axon-server/administration/multi-context.md:13]: The _admin context does not have an event store. Therefore there is no _admin directory under the data directory.

**Comment by smcvb** [comment]
Replaced by #133

**Pull request closed without merging**